### PR TITLE
Add django-feature-policy

### LINKED ISSUE #735: Consider adding django-feature-policy
https://pypi.org/project/django-feature-policy/

### PR DESCRIPTION
**Pull request by DmytroLitvinov**
* update poetry version;
* add django-feature-policy, closes #735

### CODE REVIEW

File: {{cookiecutter.project_name}}/pyproject.toml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 [build-system]
-requires = ["poetry==0.12.11"]
+requires = ["poetry==0.12.12"]
```
Comment (sobolevn) [inline {{cookiecutter.project_name}}/pyproject.toml]: Also: https://wemake-django-template.readthedocs.io/en/latest/pages/template/development.html#updating-poetry-version

And: https://github.com/wemake-services/wemake-django-template/blob/master/pyproject.toml#L2

Comment (DmytroLitvinov) [reply {{cookiecutter.project_name}}/pyproject.toml]: Nice catch.

**Comment by DmytroLitvinov** [comment]
@sobolevn I have changed already.
Could you check one more time and merge it if it is okay.

File: {{cookiecutter.project_name}}/server/settings/components/common.py [hunk 2]
```diff
@@ -54,6 +54,7 @@
 
     # Django:
     'django.middleware.security.SecurityMiddleware',
+    'django_feature_policy.FeaturePolicyMiddleware',  # django-feature-policy
```
Comment (sobolevn) [inline {{cookiecutter.project_name}}/server/settings/components/common.py]: Is it a proper place to add this middleware? Just asking

File: {{cookiecutter.project_name}}/pyproject.toml (hunk 1 above)
Comment (sobolevn) [reply {{cookiecutter.project_name}}/pyproject.toml]: One more place: https://github.com/wemake-services/wemake-django-template/blob/master/pyproject.toml#L2

Sorry for the complex process.

File: {{cookiecutter.project_name}}/server/settings/components/common.py (hunk 2 above)
Comment (DmytroLitvinov) [reply {{cookiecutter.project_name}}/server/settings/components/common.py]: Yes, this is a correct place after reading of ["Installation"](https://github.com/adamchainz/django-feature-policy#installation) section.
We should put it after `SecurityMiddleware`

File: {{cookiecutter.project_name}}/pyproject.toml (hunk 1 above)
Comment (DmytroLitvinov) [reply {{cookiecutter.project_name}}/pyproject.toml]: Force pushed update, also updated at travis file

**Comment by sobolevn** [comment]
Thanks! 

doge

**Pull request merged**